### PR TITLE
newsletter october

### PR DESCRIPTION
**Pull request by wipfli**
Newsletter for October.

### CODE REVIEW

**Review by louwers — APPROVED** [review]

File: content/news/2023-10-30-maplibre-newsletter-october-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+# MapLibre Newsletter - October 2023
+
+## MapLibre GL JS
+
+Just in time for this newsletter, MapLibre GL JS has gained over [5000 Stars on GitHub](https://github.com/maplibre/maplibre-gl-js/stargazers). Great that we got so far and thanks to everyone who contributed in the past and made this possible!
+
+In the last month, 5 releases of MapLibre GL JS were made. Below you find some highlights, for a full list of improvements please have a look at the [changelog](https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md).
+
+- Safari DEM fix: In a new version of the Safari browser, Apple has introduced some random noise in images in private mode. While usually users don't notice this effect, it had drastic consequences for terrain3d and client-side hillshading. Mike Barry fixed this issue by using a JavaScript library for decoding PNGs rather than the browser api. See [pull request](https://github.com/maplibre/maplibre-gl-js/pull/3185).
```
Comment (HarelM) [inline content/news/2023-10-30-maplibre-newsletter-october-2023/index.md:17]: Not accurate, he ended up using the browser's video frame API.

**Comment by HarelM** [comment]
I've added a minor comment, otherwise web is approved.

File: content/news/2023-10-30-maplibre-newsletter-october-2023/index.md (hunk 1 above)
Comment (wipfli) [inline content/news/2023-10-30-maplibre-newsletter-october-2023/index.md:17]:
```suggestion
- Safari DEM fix: In a new version of the Safari browser, Apple has introduced some random noise in images in private mode. While usually users don't notice this effect, it had drastic consequences for terrain3d and client-side hillshading. Mike Barry fixed this issue by using the browser's video frame API to decode PNGs. See [pull request](https://github.com/maplibre/maplibre-gl-js/pull/3185).
```

Comment (wipfli) [reply content/news/2023-10-30-maplibre-newsletter-october-2023/index.md]: Fixed

**Pull request merged**